Automattic/woocommerce-payments
# Update Checkout appearance section to display WooPay checkout preview UI

Fixes #6995 

#### Changes proposed in this Pull Request









This PR updates **WooCommerce > Settings > Payments > WooPayments > WooPay > Checkout appearance** so that the WooPay checkout preview resembles the WooPay checkout page.

#### Testing instructions





Test: Store name and logo
* As a merchant, navigate to **WooCommerce > Settings > Payments > WooPayments > WooPay > Checkout appearance**.
* Click the `Upload custom logo` button and upload one of the accepted image file types (e.g. HEIC, WEBP, SVG, PNG, or JPG).
* Ensure the preview now has the custom logo displayed.
* Delete the logo by clicking the trash icon.
* Ensure the preview now has the store's name displayed.

https://github.com/Automattic/woocommerce-payments/assets/6844516/6ba8a247-ba69-47a5-a712-985e8181d6fc


Test: Checkout privacy policy
* As a merchant, navigate to **WooCommerce > Settings > Payments > WooPayments > WooPay > Checkout appearance**.
* Add some text to the `Policies and custom text` field.
* Ensure the text changes are displayed in the preview. Also, ensure spacing is adjusted accordingly.


https://github.com/Automattic/woocommerce-payments/assets/6844516/396f180f-a2a9-4bce-9bcd-4372dff12a96


---

**Screen Caps**

Mobile
![image](https://github.com/Automattic/woocommerce-payments/assets/6844516/5c60dfe5-0633-4158-963e-c795aa95b5db)

![image](https://github.com/Automattic/woocommerce-payments/assets/6844516/1a508946-33c9-4d61-80dd-562019a2c7da)

Desktop

![image](https://github.com/Automattic/woocommerce-payments/assets/6844516/8d488777-486e-403d-93e3-81bff37a3e29)

![image](https://github.com/Automattic/woocommerce-payments/assets/6844516/b18e760e-054b-4d44-b7ca-00923b0eaa83)


-------------------


**Post merge**
